mne/annotations.py

```diff
@@ -0,0 +1,79 @@
+# Authors: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from datetime import datetime
+import time
+import json
+
+
+class Annotations(object):
+    """Annotation object for annotating segments of raw data.
+
+    Parameters
+    ----------
+    onset: array of float, shape (n_annotations,)
+        Annotation time onsets from the beginning of the recording.
+    duration: array of float, shape (n_annotations,)
+        Durations of the annotations.
+    description: array of str, shape (n_annotations,)
+        Array of strings containing description for each annotation.
+    orig_time: float | int | instance of datetime | array of int | None
+        A POSIX Timestamp, datetime or an array containing the timestamp as the
+        first element and microseconds as the second element. Determines the
+        starting time of annotation acquisition. If None (default),
+        starting time is determined from beginning of raw data.
```

Comment:
I think we need to be clearer here. For example, what should I use if my annotations start at `t=0` (synced with what we think of internally as t=0)? What if they start at `t=first_samp/sfreq` (synced with what Neuromag uses as t=0)? It's not quite clear to me how to get my Annatotaions to align properly with my raw data in those cases.